tests/core/Logger.test.ts

```
@@ -12,6 +12,80 @@ class MockTransport implements Transport {
+        level: "boring",
+        colorize: true,
+        timestamp: false,
+        transports: [formattingTransport],
+      });
+
+      logger.boring("Plain boring message");
+
+      expect(formattingTransport.outputs).toHaveLength(1);
+      expect(formattingTransport.outputs[0]).toContain("[BORING]");
+      expect(formattingTransport.outputs[0]).toContain("Plain boring message");
+      expect(formattingTransport.outputs[0]).not.toMatch(/\x1b\[[0-9;]*m/);
```

ESLint

[error] Unexpected control character(s) in regular expression: \x1b.

(no-control-regex)